spec/requests/graphql/mutations/users/create_user_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::Users::CreateUser, type: :request do
+  describe '.resolve' do
+    it 'can create a new user' do
+      post graphql_path, params: { query: query(name: 'Assassin gallant', email: 'user@email.com', birthdate: '1990-10-10') }
+      result = JSON.parse(response.body)
+
+      data = result['data']['createUser']
+
+      expect(data['id']).to_not eq(nil)
+      expect(data['id']).to be_a(String)
+      expect(data['name']).to eq('Assassin gallant')
+      expect(data['email']).to eq('user@email.com')
+      expect(data['birthdate']).to eq('1990-10-10')
+      expect(data['events']).to eq([])
+      expect(data['eras']).to eq([])
+
+      expect(User.count).to eq(1)
+    end
+  end
+
+  def query(name:, email:, birthdate:)
+    <<~GQL
+      mutation {
+        createUser(input:{
+            name: "#{name}"
+            email: "#{email}"
+            birthdate: "#{birthdate}"
+            }){
+              id
+              name
+              email
+              birthdate
+              events {
+                id
+              }
+              eras {
+                id
+              }
+            }
+          }
+    GQL
+  end
+end
```

Comment:
Lookin Solid

Reply:
Should we add a test to see what happens when the user tries to enter an email that already exists? Or is that part of the sad path testing we've been skipping because graphQL handles it?

Reply:
I had thought it would fall into the error handling that graphQL handles but lemme try it out right now

Reply:
@AngelaGuardia GraphQL does handle it nicely but I added this sad path test!

Reply:
Nice testing here, good work.